icgc-argo/argo-data-submission
# Download pyega3

To test:
```
nextflow tests/checker.nf -params-file tests/test-job-pyega3.json
```

Test should download file using Pyega3 and output file and decrypted MD5

`test-job-pyega3.json` contents:
```
{
    "project" : "TEST",
    "ega_id" : "EGAF00001770106",  ### BAM file from 1000genomes
    "cpus": 1,
    "mem": 0.5
}
```

Additional info:
```
###EGA valid credentials needed. Is preset in tests/nextflow.config
export PYEGA3_EGA_USER="ega-test-data@ebi.ac.uk"
export PYEGA3_EGA_PASS="egarocks"
```
